core/src/test/java/com/gentics/mesh/core/schema/field/DateListFieldMigrationTest.java

```diff
@@ -51,15 +48,6 @@ public void testRemove() throws Exception {
 		removeField(CREATEDATELIST, FILL, FETCH);
 	}
 
-	@Override
-	@Test
-	public void testRename() throws Exception {
```

Comment:
Check why this was removed.

Reply:
A custom script was used for renaming fields.